packages/react-dom/src/__tests__/ReactDOMServerIntegrationFragment-test.internal.js

```diff
@@ -18,7 +18,6 @@ let ReactDOMServer;
 function initModules() {
   // Reset warning cache.
   jest.resetModuleRegistry();
-  require('shared/ReactFeatureFlags').enableReactFragment = true;
```

Comment:
Since you removed the last internal import in this file, we should also remove `.internal.js` prefix. Then tests will run in production env too :-)

Reply:
https://github.com/facebook/react/pull/11673